sqlmesh/migrations/v0065_add_auto_restatements.py

```diff
@@ -0,0 +1,46 @@
+"""Add the auto restatements table."""
+
+from sqlglot import exp
+
+from sqlmesh.utils.migration import index_text_type
+
+
+def migrate(state_sync, **kwargs):  # type: ignore
+    engine_adapter = state_sync.engine_adapter
+    schema = state_sync.schema
+    auto_restatements_table = "_auto_restatements"
```

Comment:
this entire table is only going to store auto restatements? i could see us storing other things on the snapshot version level

Reply:
Yes, I can see that too. When the time comes we can rename it. Today, I don't have other use cases for it.

Reply:
Just a heads up that there's another 0065 script [here](https://github.com/TobikoData/sqlmesh/pull/3512).

Reply:
thanks!